include/ConformalTracking.h

```diff
@@ -64,44 +64,44 @@ class ConformalTracking : public Processor {
   void getCollection(LCCollection*&, std::string, LCEvent*);
 
   // Plotting function for displaying cells
-  void drawline(KDCluster*, KDCluster*, int, int style = 1);
+  void drawline(SKDCluster const&, SKDCluster const&, int, int style = 1);
 
   // Pattern recognition algorithms:
 
   // Cell creation
-  KDCluster* extrapolateCell(Cell*, double);
-  void       extendSeedCells(std::vector<Cell*>&, KDTree*, bool, const std::vector<KDCluster*>&);
+  SKDCluster extrapolateCell(Cell::SCell const&, double);
```

Comment:
what is the purpose of the const&? The cell that is pointed to will be modified during this function

Reply:
nevermind, thats not true...